openedx/core/djangoapps/user_api/api.py

```diff
@@ -92,16 +92,33 @@ def get_login_session_form():
         platform_name=configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME)
     )
 
+    email = ''
+    email_field_restrictions = {
+        "min_length": accounts.EMAIL_MIN_LENGTH,
+        "max_length": accounts.EMAIL_MAX_LENGTH,
+    }
+
+    if third_party_auth.is_enabled():
```

Comment:
It would be better to move this logic to a function similar to what is done [here](https://github.com/edx/edx-platform/blob/1fa5c38ae087330e8112a1eb887f37a8c73fc0fc/openedx/core/djangoapps/user_api/api.py#L229) I think [_apply_third_party_auth_overrides](https://github.com/edx/edx-platform/blob/1fa5c38ae087330e8112a1eb887f37a8c73fc0fc/openedx/core/djangoapps/user_api/api.py#L834) is a good name here too.